src/pybind/mgr/cephadm/module.py

```diff
@@ -1992,6 +1992,7 @@ def plan(self, specs: List[GenericSpec]) -> List:
         return results
 
     def _apply_service_spec(self, spec: ServiceSpec) -> str:
+        used_default_placement = spec.placement.is_empty()
```

Comment:
this var 'used_default_placement' doesnt seem to used after its create.

Reply:
fixed